### PR TITLE
JBTM-3658 Remove LRA coordinator Quarkus module

### PR DESCRIPTION
**Pull request by xstefank**
As it was moved to a separate repository.

https://issues.redhat.com/browse/JBTM-3658

!CORE !TOMCAT !AS_TESTS !RTS !JACOCO !XTS !QA_JTA !QA_JTS_OPENJDKORB !PERF LRA !DB_TESTS !MySQL !db2 !postgres !oracle

### CODE REVIEW

**Review by mmusgrov — COMMENTED** [review]
How does this PR resolve the related topic that we discussed on zulip (https://narayana.zulipchat.com/#narrow/stream/323715-developers/topic/How.20to.20run.20an.20LRA.20Coordinator.20using.20Quarkus): there has been no progress on that topic for two weeks.

**Comment by mmusgrov** [comment]
I'm holding this one pending resolution of my last comment and also pending the work on the quickstart repo which has a reliance on this module.

File: rts/lra/test/pom.xml [hunk 1]
```diff
@@ -47,11 +45,17 @@
     <lra.coordinator.port>8080</lra.coordinator.port>
```
Comment (jmfinelli) [inline rts/lra/test/pom.xml:45]: I think that we can remove the properties from line 39 to 44, WDYT?

File: rts/lra/test/pom.xml [hunk 2]
```diff
@@ -47,11 +45,17 @@
...
     <!-- the property used by NarayanaLRAClient when passed into the container as the JVM parameter -->
     <lra.coordinator.url>http://${lra.coordinator.host}:${lra.coordinator.port}/lra-coordinator</lra.coordinator.url>
+
+    <!-- The following filename is used for starting the WildFly instance that hosts the LRA participant; this file is a copy of the
+     standalone.xml file provided in WildFly (${env.JBOSS_HOME}/standalone/configuration/standalone.xml). Please, see the plugins
+     section of this profile (specifically, maven-antrun-plugin) for further info -->
+    <lra.participant.xml.filename>lra-participant.xml</lra.participant.xml.filename>
```
Comment (jmfinelli) [inline rts/lra/test/pom.xml:52]: Why have you moved this here? This property is only used in the profile `arq`

File: rts/lra/test/pom.xml [hunk 3]
```diff
@@ -47,11 +45,17 @@
...
     <!-- for test executions where coordinator is started as a separate JVM (e.g. for Quarkus) and the test application
              is deployed at the different JVM; these properties define where the test JVM application is started -->
     <test.application.host>localhost</test.application.host>
     <test.application.port>8180</test.application.port>
-
+    <!-- this property is only used in direct submodules, if used outside this scope it needs to be redined -->
+    <wildfly.trace.logging.cli.script>${project.basedir}/../wildfly-trace-logging.cli</wildfly.trace.logging.cli.script>
```
Comment (jmfinelli) [inline rts/lra/test/pom.xml:58]: Why have you moved this here? If I understand correctly, this property is only used in the profile `trace.lra`

**Review by jmfinelli — CHANGES_REQUESTED** [review]

**Review by mmusgrov — CHANGES_REQUESTED** [review]
The changes that remove the `coordinator-quarkus` module look good.

But the PR contain a lot of other changes relating to testing on wildfly that are unrelated - ie it's not clear to me how those relate to the issue description. Would it be possible to raise those changes in a separate PR so that they can be reviewed independently.

I am going to remove the Hold label while we discuss this PR and add the `Urgent` label since we need this PR for the next release of Narayana.

**Comment by mmusgrov** [comment]
I removed the urgent label since the PR marco referenced means that we just need to disable the deployment of the `lra-coordinator-quarkus` artifact. Then we can avoid issues with deploying the same artifact as the one deployed by the `https://github.com/jbosstm/lra-coordinator-quarkus` repository.

**Comment by xstefank** [comment]
Still on PTO but I'll fix this if it's urgent. @mmusgrov the test changes are related to the move to the WFLY deployed coordinator where we still passed for instance logging info for the quarkus which wouldn't work. So @mmusgrov and @jmfinelli should I clear this PR to only remove the module and we will clean up testsuite in follow-up?

**Comment by marcosgopen** [comment]
Thanks @xstefank for your answer! Michael removed the urgent label so we can talk about it when you will be back from PTO.
Thanks again for your PR!

**Review by jmfinelli — APPROVED** [review]
LGTM! Thank you very much @xstefank !

**Review by mmusgrov — APPROVED** [review]
Thanks Martin for separating out the commits. I think this one can now be merged.

**Pull request merged**